src/quo2/components/list_items/channel/style.cljs

```diff
@@ -0,0 +1,42 @@
+(ns quo2.components.list-items.channel.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(def ^:private label-colors
+  {:notification {:light colors/neutral-100
```

Comment:
`colors/theme-colors` can be used here to avoid mentioning `:dark`, `:light` directly